modules/data.land/R/dataone_download.R

```diff
@@ -12,6 +12,8 @@
 #' @export
 #'
 
+#' @examples doi_download(id = "doi:10.6073/pasta/63ad7159306bc031520f09b2faefcf87", filepath = "/fs/data1/pecan.data/dbfiles/")
```

Comment:
This probably need to be labeled as not run, otherwise it might interfere with the package build since most systems won't have that specific filepath.